test/test-network/systemd-networkd-tests.py

```diff
@@ -3481,6 +3501,54 @@ def test_ipv6_proxy_ndp(self):
         for i in range(1, 5):
             self.assertRegex(output, f'2607:5300:203:5215:{i}::1 *proxy')
 
+    def test_ipv6_neigh_retrans_time(self):
+        link='test25'
+        copy_network_unit('25-dummy.netdev', '25-dummy.network')
+        start_networkd()
+
+        self.wait_online([f'{link}:degraded'])
+        remove_network_unit('25-dummy.network')
+
+        # expect retrans_time_ms updated
+        copy_network_unit('25-ipv6-neigh-retrans-time-3s.network')
+        networkctl_reload()
```

Comment:
Please always call `wait_online()` after `networkctl_reload()`, otherwise the setting may not be assigned yet, and the check below may fail.

Reply:
Done.